sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonCreateForm.java

```diff
@@ -286,9 +292,9 @@ protected Button createPersonSearchButton(String personSearchLoc) {
 					SimilarPersonDto pickedPerson = personSearchField.getValue();
 					if (pickedPerson != null) {
 						// add consumer
-						searchedPerson = FacadeProvider.getPersonFacade().getByUuid(pickedPerson.getUuid());
-						setPerson(searchedPerson);
-						enablePersonFields(false);
+						person = FacadeProvider.getPersonFacade().getByUuid(pickedPerson.getUuid());
+						setPerson(person);
+						enablePersonFields(false, true);
```

Comment:
when a person is picked then all fields are disabled no matter if they are empty or not. This does not satisfy the comment added by @adinaflorea9 in the ticket.

Reply:
I think Adina's comment is out of scope for this issue because it would involve a lot more changes (e.g. this would need to be done for all create forms that include person details, not only the case create form). I've created a follow-up issue for it: https://github.com/hzi-braunschweig/SORMAS-Project/issues/11781